### PR TITLE
Add aggregation section (collection stages) to the docs

### PR DESCRIPTION
**Pull request by rumyantseva**
# Description

Refs #1399.

## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [ ] I ran `task all`, and it passed.
* [ ] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [ ] I checked comments rendering with `task godocs`.
* [x] I ensured that the title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

**Comment by rumyantseva** [comment]
@AlekSi PTAL!

File: website/docs/reference/supported_commands.md [hunk 1]
```diff
@@ -11,3 +11,51 @@ sidebar_position: 1
...
 |                   | `comment`               | ⚠️     | Ignored                                                   |
 |                   | `authorizedCollections` | ⚠️     | Ignored                                                   |
+
+## Aggregation pipelines
+
+The epic - [Issue](https://github.com/FerretDB/FerretDB/issues/9).
+
+| Command     | Argument             | Status | Comments                                                  |
+|-------------|----------------------|--------|-----------------------------------------------------------|
+| `aggregate` |                      | ❌      | [Issue](https://github.com/FerretDB/FerretDB/issues/1410) |
```
Comment (Fashander) [inline website/docs/reference/supported_commands.md:21]: Won't it be better to have this placed together with the other commands above?

**Review by AlekSi — APPROVED** [review]

File: website/docs/reference/supported_commands.md (hunk 1 above)
Comment (Fashander) [reply website/docs/reference/supported_commands.md:21]: Good idea too. @AlekSi can take a look?

**Review by Fashander — APPROVED** [review]
LGTM

File: website/docs/reference/supported_commands.md (hunk 1 above)
Comment (AlekSi) [reply website/docs/reference/supported_commands.md:21]: We could do it later if needed

**Pull request merged**